packages/calcite-components/src/components/action/action.scss

```diff
@@ -46,6 +77,35 @@
 
   background-color: var(--calcite-action-background-color, var(--calcite-color-foreground-1));
   color: var(--calcite-action-text-color, var(--calcite-color-text-3));
+
+  border-start-start-radius: var(
```

Comment:
There's some duplication here. Can you DRY it up by adding a rule for these props that targets both `:host` and `.button`?